### ISSUE TITLE
Allow to manually execute deviceready event

### ISSUE THREAD

**Issue by gregd** [katzer/cordova-plugin-local-notifications#1444]
Ionic 3 has lazy loading which seems to break Cordova deviceready callback order. I posted the bug here https://github.com/ionic-team/ionic/issues/13525 It seems that Ionic `platform.ready().then(() => { ... }` is called after LocalNotifications `deviceready` callback. It would be good to have a work around to manually trigger below code from `platform.ready().then(...)` callback. 
```
// Called after 'deviceready' event
channel.deviceready.subscribe(function () {
    // Device is ready now, the listeners are registered
    // and all queued events can be executed.
    exec(null, null, 'LocalNotification', 'deviceready', []);
});
```

I guess this can be done with plugin configuration option and with new LocalNotification method. The method should be called after event handlers like `click` are registered.

## Your Environment

* Plugin version: 0.8.5
* Platform: android 
* OS version: Android 7/8
* Device manufacturer / model: Google
* Cordova version (```cordova -v```): 7.1.0
* Cordova platform version (```cordova platform ls```): android 6.3.0
* Plugin config -
* Ionic Version (if using Ionic) 3.9.2 

## Steps to Reproduce
Described here https://github.com/ionic-team/ionic/issues/13525

**Comment by katzer** [comment]
In future its possible by defining `skipLocalNotificationReady`.

```js
window.skipLocalNotificationReady = true
```

Once the app and Ionic is ready, you can fire the queued events manually.

```js
cordova.plugins.notification.local.fireQueuedEvents();
```

**Comment by rapoell** [comment]
Hi Sebastián,

I used the version with the commit of Feb 5 (using skipLocalNotificationReady and fireQueuedEvents()) in iOS 11.2 simulator and iOS 10.3.3 on a physical device (iPhone5 ). In both cases when I start the app from a notification, the startup screen appears and then, after a second or so, the screen turns black, the app crashes apparently. After approx. 10 seconds the screen returns to one of the home/app pages of iOS from where I selected the notification.
It doesn't matter if I call the related code or comment them, the behavior is identical.
Reverting to 0.9.0-beta.2 solves the black-screen issue.

Any idea where this might come from?
